### PR TITLE
`web-ext` can be installed with `brew`

### PR DESCRIPTION
**Pull request by abitrolly**

### CODE REVIEW

File: src/content/documentation/develop/getting-started-with-web-ext.md [hunk 1]
```diff
@@ -53,7 +53,15 @@ date: 2021-01-27 23:47:23
 
 ## Installation
 
-`web-ext` is a node-based application that you install with the [nodejs/npm](https://nodejs.org/) tool. Install web-ext using the following command:
+`web-ext` is a node-based application than can be installed it with `brew` or natively with [nodejs/npm](https://nodejs.org/) tool.
```
Comment (rebloor) [inline src/content/documentation/develop/getting-started-with-web-ext.md:56]:
```suggestion
`web-ext` is a node-based application. You install it with `brew` or the [nodejs](https://nodejs.org/) npm tool.
```

File: src/content/documentation/develop/getting-started-with-web-ext.md [hunk 2]
```diff
@@ -53,7 +53,15 @@ date: 2021-01-27 23:47:23
...
+
+Installing with `brew`:
```
Comment (rebloor) [inline src/content/documentation/develop/getting-started-with-web-ext.md:58]:
```suggestion
Install with `brew` using:
```

File: src/content/documentation/develop/getting-started-with-web-ext.md [hunk 3]
```diff
@@ -53,7 +53,15 @@ date: 2021-01-27 23:47:23
...
+
+```shell
+brew install web-ext
+```
+
+Installing with `npm`:
```
Comment (rebloor) [inline src/content/documentation/develop/getting-started-with-web-ext.md:64]:
```suggestion
Install with `npm` using:
```

**Review by rebloor — CHANGES_REQUESTED** [review]
@abitrolly Thanks for spotting this omission

**Comment by abitrolly** [comment]
Thanks for the suggestions. Applied.

**Review by rebloor — APPROVED** [review]

**Pull request merged**